packages/translations/locales/fr/admin.json

```
@@ -1540,5 +1541,135 @@
+      "extend": {
+        "title": "Prolonger l'abonnement",
+        "description": "Prolonger la période d'abonnement de jours supplémentaires.",
+        "additionalDays": "Jours supplémentaires"
+      },
+      "upgrade": {
+        "title": "Mettre à niveau l'abonnement",
+        "description": "Passer à un forfait supérieur."
+      },
+      "downgrade": {
+        "title": "Rétrograder l'abonnement",
+        "description": "Passer à un forfait inférieur (prend effet à la fin de la période)."
```

Biome (2.1.2)

[error] The key cancel was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)